Source/Scene/Instanced3DModel3DTileContent.js

```diff
@@ -419,8 +419,9 @@ define([
             instanceScale.x = 1.0;
             instanceScale.y = 1.0;
             instanceScale.z = 1.0;
-            var scale = featureTable.getProperty('SCALE', i, ComponentDatatype.FLOAT);
+            var scale = featureTable.getProperty('SCALE', i, ComponentDatatype.FLOAT, 1);
             if (defined(scale)) {
+                if (Array.isArray(scale)) scale = scale[0];
```

Comment:
This change will probably need more discussion from the spec side, could you open an issue there? Using the same name SCALE to refer to per-instance scale and global scale seems prone to errors.